DuggaSys/microservices/shared_microservices/getUid_ms.php

```diff
@@ -11,7 +11,7 @@ function getUid(){
         if(isset($_SESSION['uid'])){
             $userid=$_SESSION['uid'];
         }else{
-            $userid="1";
+            $userid="guests";
```

Comment:
I am a bit unsure of this change, some services require `1`. Since this issue is not related to this microservice, should this really be altered? Maybe it is needed for the MOTD microservice?

Reply:
the problem is that there already exist a user with userid 1, and his a super user

![image](https://github.com/HGustavs/LenaSYS/assets/102600391/736fc41e-377c-4b95-a00c-516d57a5ac9c)